### ISSUE TITLE
Unable to (Fresh) install Go 1.4 to install Go 1.6 via GVM on OSX

### ISSUE THREAD

**Issue by bmurtagh** [moovweb/gvm#193]
Hello,

I'm starting my journey to learn Go, but having trouble installing Go 1.4 via GVM. This is on OSX Mavericks and I've never installed Go or GVM on the machine before. 

I've installed Mercurial, bzr via homebrew and installed Xcode Command Line Tools via ``xcode-select --install``.

Here is the error from /Users/brendan/.gvm/logs/go-go1.4-compile.log:
```
# Building C bootstrap tool.
cmd/dist

# Building compilers and Go bootstrap tool for host, darwin/amd64.
lib9
libbio
liblink
cmd/cc
cmd/gc
cmd/6l
cmd/6a
cmd/6c
/Users/brendan/.gvm/gos/go1.4/src/cmd/6c/txt.c:995:28: error: shifting a negative signed value is undefined [-Werror,-Wshift-negative-value]
/Users/brendan/.gvm/gos/go1.4/src/cmd/6c/txt.c:1045:28: error: shifting a negative signed value is undefined [-Werror,-Wshift-negative-value]
go tool dist: FAILED: clang -Wall -Wstrict-prototypes -Wextra -Wunused -Wno-sign-compare -Wno-missing-braces -Wno-parentheses -Wno-unknown-pragmas -Wno-switch -Wno-comment -Wno-missing-field-initializers -Werror -fno-common -ggdb -pipe -Wuninitialized -O2 -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -mmacosx-version-min=10.6 -c -m64 -I /Users/brendan/.gvm/gos/go1.4/include -I /Users/brendan/.gvm/gos/go1.4/src/cmd/6c -o /Users/brendan/.gvm/gos/go1.4/pkg/obj/cmd/6c/txt.o /Users/brendan/.gvm/gos/go1.4/src/cmd/6c/txt.c
```

Here are my steps:

```
[brendan@dirka ~]$ gvm version
Go Version Manager v1.0.22 installed at /Users/brendan/.gvm
```
```
[brendan@dirka ~]$ gvm list

gvm gos (installed)

```
```
[brendan@dirka ~]$ gvm install go1.4
Installing go1.4...
 * Compiling...
ERROR: Failed to compile. Check the logs at /Users/brendan/.gvm/logs/go-go1.4-compile.log
ERROR: Failed to use installed version
```

**Comment by bmurtagh** [comment]
I answered this issue by following the steps in #176. Please merge that PR ASAP to help others in my situation

**Comment by jbussdieker** [comment]
Merged #176

**Comment by tejasmanohar** [comment]
has this been released, still facing error
